ci.nix

```diff
@@ -42,9 +40,11 @@
     # We need to use the actual nixpkgs version we're working with here, since the values
     # of 'lib.systems.examples' are not understood between all versions
     let lib = nixpkgs.lib;
-    in lib.optionalAttrs (nixpkgsName == "R2003" && (__elem compiler-nix-name ["ghc865" "ghc884"])) {
+    in lib.optionalAttrs (nixpkgsName == "R2009" && (__elem compiler-nix-name ["ghc865" "ghc884"])) {
```

Comment:
Maybe we should flip this and only run ghcjs tests on a particular chosen GHC version. IDK.

Reply:
I also can't read here (for some reason I thought this was `!elem`)